src/assets/main.css

```diff
@@ -74,6 +75,15 @@
     border-top: none;
 }
 
+.icon-display {
```

Comment:
While I see the intent of this, in effect its stretching a resized image, making it look blurry.


I will fix this in the canvas itself.